great_expectations/cli/docs.py

```diff
@@ -84,35 +84,26 @@ def docs_list(ctx):
 @click.option(
     "--all",
     "-a",
+    "all_sites",
     is_flag=True,
     help="With this, all sites will get their data docs cleaned out. See data_docs section in great_expectations.yml",
 )
 @click.pass_context
-def clean_data_docs(ctx, site_name=None, all=None):
+def clean_data_docs(ctx, site_name=None, all_sites=False):
     """Delete data docs"""
-    display_not_implemented_message_and_exit()
     context = ctx.obj.data_context
-    failed = True
-    if site_name is None and all is None:
-        cli_message(
-            "<red>{}</red>".format(
-                "Please specify --all to remove all sites or specify a specific site using "
-                "--site_name"
-            )
-        )
-        sys.exit(1)
-    context.clean_data_docs(site_name=site_name)
-    failed = False
-    if not failed and context is not None:
-        toolkit.send_usage_message(
-            data_context=context, event="cli.docs.clean", success=True
-        )
-        cli_message("<green>{}</green>".format("Cleaned data docs"))
 
-    if failed and context is not None:
-        toolkit.send_usage_message(
-            data_context=context, event="cli.docs.clean", success=False
+    if site_name is None and all_sites is False:
+        toolkit.exit_with_failure_message_and_stats(
+            context,
+            usage_event="cli.docs.clean",
+            message="<red>Please specify --all to remove all sites or specify a specific site using --site_name</red>",
         )
+    context.clean_data_docs(site_name=site_name)
```

Comment:
Interesting - so if site_name is specified then only that site is cleaned, and if no site name is specified and all_sites is True then `context.clean_data_docs()` loops through all of them. This isn't super hard to figure out but it is slightly confusing as to why there is no path for all_sites at first - what do you think about adding a tiny note like this:
```suggestion
    # if site_name is None, context.clean_data_docs(site_name=site_name) will clean all sites.
    context.clean_data_docs(site_name=site_name)
```